about/pricing/comparison.md

```diff
@@ -0,0 +1,67 @@
+# Comparing pricing to similar services
+
+This page is a guide to 2i2c's services, our pricing, and how this compares with similar kinds of offerings.
+
+:::{tip}
+The content on this page can be re-used as a part of "price reasonableness and comparisons" forms when completing contracting for communities.
+:::
+
+## How to think about 2i2c's service and pricing
+
+As a non-profit, we choose our prices as a function of our estimated costs.
```

Comment:
I would say it is not only the estimated cost... I would argue that function should also consider a sustainable path forward (and the cost associated with that concept).

Reply:
Yep, I think having all the above details is maybe too much information... just putting the "sustainable" concept in the reader's head might be enough for now...

Reply:
OK, will improve this in the next commit and we can iterate in a new thread if need be!